msvc/VS2015/leela-zero.vcxproj

```diff
@@ -121,9 +122,11 @@
     <ClInclude Include="..\..\src\Training.h" />
     <ClInclude Include="..\..\src\Tuner.h" />
     <ClInclude Include="..\..\src\UCTNode.h" />
+    <ClInclude Include="..\..\src\UCTNodePointer.h" />
     <ClInclude Include="..\..\src\UCTSearch.h" />
     <ClInclude Include="..\..\src\Utils.h" />
     <ClInclude Include="..\..\src\Zobrist.h" />
+    <ClInclude Include="Header.h" />
```

Comment:
There's no Header.h in our project?

Reply:
It looks like VS somehow copied Header.h from somewhere; the file is dated 9/20/2015 6:20PM with sole content `#pragma once`. This only happened once (on /next a few commits ago) and I can't reproduce it now, so I'll just delete this line.